ci-operator/step-registry/ipi/conf/azurestack/creds/ipi-conf-azurestack-creds-commands.sh

```diff
@@ -62,7 +73,7 @@ do
       continue
   fi
 
-  filename=manifest_${SECRET_NAMESPACE}_secret.yml
+  filename=manifest_${SECRET_NAMESPACE}_${SECRET_NAME}_secret.yml
```

Comment:
Nit: since this seems to be fixing an existing "bug", it would be nice if it was in its own commit.

Reply:
During development for ipi ash on wwt, we hit the bug, if not fix it together, in this PR, we are not confident with the new cluster profile integration.

Reply:
@jianlinliu I was just suggesting a new commit on *this* PR, not a separate PR. But maybe I misunderstood and this is just a "bug" when WWT is introduced.

Reply:
@r4f4 get your point, I submitted the new commit for this change. This is not introduced by WWT, but the common issue for ipi azurestack (see [failure job](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_installer/6684/pull-ci-openshift-installer-master-e2e-azurestack/1605611843617820672)). I found that two csi driver secrets under same namespace created into one manifest file, during installation, only one secret created in cluster, another one is missing, and result in installation failed finally.

Could you help to review the update？

Reply:
I like it. The intention behind the change is very clear now!